csproj.in

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
If we are to require external dependencies, I would highly recommend [`jinja2`](http://jinja.pocoo.org/) for templating.

Reply:
I went with the simple string templating to avoid the hassle of jinja2. It works well for the simple replacement that's going on here.